iOS/DuckDuckGo-iOS.xcodeproj/project.pbxproj

```diff
@@ -13070,7 +13070,7 @@
 				ENABLE_USER_SCRIPT_SANDBOXING = YES;
 				GCC_C_LANGUAGE_STANDARD = gnu17;
 				GENERATE_INFOPLIST_FILE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 18.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 15.6;
```

Comment:
@federicocappelli is this expected? Should it be saying 18.6? Or it’s unrelated to the workflow updates?